opm/parser/share/keywords/000_Eclipse100/O/OPERNUM

```diff
@@ -0,0 +1,5 @@
+{
+ "name" : "OPERNUM",
```

Comment:
I'm confused: doesn't the presence of this JSON file mean that the OPERNUM *keyword* is supported. (though the corresponding *grid property* is still missing, as far as I can see.)